doc/man7/ssl.pod

```diff
@@ -89,6 +89,12 @@ includes both more private SSL headers and headers from the B<crypto> library.
 Whenever you need hard-core details on the internals of the SSL API, look
 inside this header file.
 
+ssl.h includes a macro OPENSSL_VERSION_AT_LEAST(major,minor,fix), which can be
```

Comment:
Please change `ssl.h` to `opensslv.h`

Reply:
Are users expected to include opensslv.h directly?  That's why I wrote ssl.h. Shall I change it to:

    opensslv.h (included by ssl.h) contains a macro ...

Reply:
Actually, since this is under the `ssl.h` bullet anyway, there's no need to mention it again, so I'd suggest something like this:

`OPENSSL_VERSION_AT_LEAST(major,minor,fix) is a macro which can be used in C<#if> statements in order to determine which version of the library is being used. This can be used to either enable optional features at compile time, or work around issues with a previous version. See L<OPENSSL_VERSION_NUMBER(3)>.`